### PR TITLE
Add support to provide config to the root

### PR DESCRIPTION
**Pull request by eneufeld**
- add config input
- use two way databinding in example for data

### CODE REVIEW

File: packages/angular-material/example/app/app.component.ts [hunk 1]
```diff
@@ -71,14 +71,14 @@ const itemTester: UISchemaTester = (_schema, schemaPath, _path) => {
       </button>
     </div>
     <jsonforms
-      [data]="selectedExample.data"
+      [(data)]="selectedExample.data"
```
Comment (sdirix) [inline packages/angular-material/example/app/app.component.ts:74]: Do we want two-way binding here?

File: packages/angular/src/jsonforms-root.component.ts [hunk 2]
```diff
@@ -63,7 +64,9 @@ export class JsonForms implements OnChanges, OnInit {
             },
             uischemas: this.uischemas,
             i18n: { locale: this.locale, localizedSchemas: undefined, localizedUISchemas: undefined },
-            renderers: this.renderers
+            renderers: this.renderers,
+            config: this.config,
+            readonly: this.readonly
```
Comment (sdirix) [inline packages/angular/src/jsonforms-root.component.ts:69]: good catch

**Review by sdirix — APPROVED** [review]
LGTM

File: packages/angular-material/example/app/app.component.ts (hunk 1 above)
Comment (eneufeld) [reply packages/angular-material/example/app/app.component.ts:74]: yes, to showcase that the data is updated, we show this value in a div above the form

**Pull request merged**